mocha.config.js

```diff
@@ -1,5 +1,7 @@
+const path = require('path');
+
 module.exports = {
-  require: '@babel/register',
+  require: path.join(__dirname, '.mocha/babel-require.js'),
```

Comment:
could we do the `.mocha/babel-require.js` right in this file and then let it require itself?

Reply:
Not sure what you mean. Had to extract the whole require part to babel-require.js because I have to pass options to babel. Found no other way to do that